### ISSUE TITLE
Don't require a glossary

### ISSUE THREAD

**Issue by xeruf** [openpatch/hyperbook#292]
On a new project, I get a build error if the `glossary` folder is missing:
```sh
npx hyperbook build
info  - No Archives found

> hyperbook-simple-template@0.1.1 build
> next build && next export

info  - Skipping validation of types
info  - Linting  
info  - Creating an optimized production build  
info  - Compiled successfully
info  - Collecting page data .Error: ENOENT: no such file or directory, scandir '/home/janek/data/1-projects/ftt/about/.hyperbook/glossary'
    at Object.readdirSync (fs.js:1048:3)
    at getAllFiles (/home/janek/data/1-projects/ftt/about/.hyperbook/.next/server/chunks/587.js:523:61)
    at getStaticPaths (/home/janek/data/1-projects/ftt/about/.hyperbook/.next/server/pages/glossary/[...term].js:125:85)
    at buildStaticPaths (/home/janek/data/1-projects/ftt/about/.hyperbook/node_modules/next/dist/build/utils.js:592:39)
    at /home/janek/data/1-projects/ftt/about/.hyperbook/node_modules/next/dist/build/utils.js:886:121
    at async Span.traceAsyncFn (/home/janek/data/1-projects/ftt/about/.hyperbook/node_modules/next/dist/trace/trace.js:79:20) {
  errno: -2,
  syscall: 'scandir',
  code: 'ENOENT',
  path: '/home/janek/data/1-projects/ftt/about/.hyperbook/glossary'
}

> Build error occurred
Error: Failed to collect page data for /glossary/[...term]
    at /home/janek/data/1-projects/ftt/about/.hyperbook/node_modules/next/dist/build/utils.js:916:15 {
  type: 'Error'
}
i```

**Comment by mikebarkmin** [comment]
This should not happen. I will look into it.

**Comment by xeruf** [comment]
Using latest version I still get this error :/
```sh
❯ npx hyperbook --version
0.8.4
❯ npx hyperbook build
[FTT Handbuch] Building Book.
[FTT Handbuch] No archives found.
[FTT Handbuch] Exporting HTML files...[FTT Handbuch] [Error: ENOENT: no such file or directory, scandir '/home/janek/data/1-projects/ftt/handbook/.hyperbook/glossary'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'scandir',
  path: '/home/janek/data/1-projects/ftt/handbook/.hyperbook/glossary'
}

[FTT Handbuch] 
> Build error occurred

[FTT Handbuch] Error: Failed to collect page data for /glossary/[...term]
    at /home/janek/data/1-projects/ftt/handbook/.hyperbook/node_modules/next/dist/build/utils.js:916:15 {
  type: 'Error'
}
```

**Comment by mikebarkmin** [comment]
This should be fixed in hyperbook@0.9.1.
